README.md

```diff
@@ -1,46 +1,15 @@
 # target-intacct
+A custom Meltano target that sends data into Intacct.
+- The target is based on HotGlueXYZ's [Target-Intacct](https://github.com/hotgluexyz/target-intacct)
 
-This is a [hotglue](https://hotglue.xyz) target that sends CSV data back to Sage Intacct.
 
 ## Quick Start
 
 1. Install
 
-    ```bash
-    pip install git+https://github.com/hotgluexyz/target-intacct.git
-    ```
+   - Clone the Repository 
 
-2. Create the config file
+    - In the root folder run `pip install .`
```

Comment:
Remove the space before the period. Move the closing back tick to before the period.

Reply:
I wonder if the "." is indicating to install the current directory.  For local file system installs it checks the directory that is provided and sees if it has a setup.py.